File: HackerRank/30-Days-of-Code/C++/day00-HelloWorld.cpp
#include <iostream>
#include <string>
using namespace std;
//Question link:https://www.hackerrank.com/challenges/30-hello-world/problem?isFullScreen=true

int main() {
    string input_string;
    getline(cin,input_string);
    cout << "Hello, World." << endl;
    cout << input_string << endl;
       
    return 0;
    //Sample output
    /*Hello, World. 
    Welcome to 30 Days of Code!*/
}

File: HackerRank/30-Days-of-Code/Java/day00HelloWorld.java
import java.util.Scanner;

//Question link:https://www.hackerrank.com/challenges/30-hello-world/problem?isFullScreen=true

public class day00HelloWorld {
	public static void main(String[] args) {
        
		Scanner scan = new Scanner(System.in); 
		
		
		String inputString = scan.nextLine(); 


		scan.close(); 
      
		
		System.out.println("Hello, World.");
        System.out.println(inputString);
        /*Sample output
        Hello, World. 
        Welcome to 30 Days of Code!*/
	}
}